src/core/generateZodSchema.test.ts

```diff
@@ -950,26 +950,26 @@ describe("generateZodSchema", () => {
            *
            * @format ipv4
            */
-          ipv4: z.string().ip({ version: "v4" }),
+          ipv4: z.string().ipv4({ version: "v4" }),
           /**
            * The hero's ipv6 address.
            *
            * @format ipv6
            */
-          ipv6: z.string().ip({ version: "v6" }),
+          ipv6: z.string().ipv6({ version: "v6" }),
           /**
            * The hero's ip address.
            *
            * @format ip
            */
-          ip: z.string().ip(),
+          ip: z.string().ipv4(),
```

Comment:
```suggestion
          ip: z.ipv4(),
```

This format is deprecated

<img width="1487" height="1271" alt="image" src="https://github.com/user-attachments/assets/7f0f6821-6ae8-48d5-8c70-d326dcdda3ac" />